app/static/cursor.js

```diff
@@ -0,0 +1,31 @@
+const CURSORS_PATH = "/media/cursors/";
+window.addEventListener("load", get_cursor_choices);
+
+/**
+ * @summary This function gets cursor choices from the db.
+ */
+function get_cursor_choices() {
+  const request = new XMLHttpRequest();
+  request.open("GET", "/cursor/load", true);
+  request.onload = change_cursor;
+  request.send();
+}
+
+/**
+ * @summary This function changes the primary cursor and the secondary
+ * cursor according to users' choices.
+ */
+function change_cursor() {
+  const cursor_settings = JSON.parse(JSON.parse(this.response));
```

Comment:
Do we have an idea about what happens if we dynamically add links to the page, and how to make them change the style as well?

Reply:
I didn't add links, just changed their style, and it works amazing :)

Reply:
The question is what happens if some other developer add links after your script runs

Reply:
This case is now handled as well :)